pantry-finder-api/spec/serializers/foodbanks_serializer_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
```

Comment:
The file name should be `foodbank_serializer_spec.rb` instead of the plural `foodbanks_serializer_spec.rb` to match the class name.